src/ngResource/resource.js

```diff
@@ -575,7 +575,11 @@ angular.module('ngResource', ['ng']).
 
         forEach(actions, function(action, name) {
           var hasBody = /^(POST|PUT|PATCH)$/i.test(action.method);
-          var cancellable = false;
+          var cancellable = angular.isDefined(action.cancellable) ? action.cancellable :
+            (options && angular.isDefined(options.cancellable)) ? options.cancellable :
+              provider.defaults.cancellable;
+
+          var simulatedTimeout;
```

Comment:
`simulatedTimeout` is a little confusing imo (what is "simulated" about it).
<sub>(On the other hand, I'm notiriously bad with names :smiley:)</sub>